api-example.c

```diff
@@ -8,7 +8,7 @@
  */
 
 /* Compile:
- *   gcc api-example.c -Icompat/jansson-2.6/src -Icompat/libusb-1.0/libusb -o cgminer-api
+ *   gcc api-example.c -Icompat/jansson-2.9/src `pkg-config --cflags libusb-1.0` -o cgminer-api
```

Comment:
This doesn't seem to work on OSX, but we should be able to remove libusb https://github.com/ckolivas/cgminer/pull/718